torchdata/dataloader2/utils/worker.py

```diff
@@ -84,9 +153,22 @@ def process_reset_fn(
     reset the random state of the ``DataPipe`` graph and the global random states for ``torch``,
     ``random`` and ``numpy``.
     """
+    # Reset non-sharding process first
+    graph = traverse_dps(datapipe)
+    dispatch_process_dps = find_dps(graph, communication.iter._IterateQueueDataPipes)
```

Comment:
so... this is is on worker process? -- the `_IterateQueueDataPipes` is really the "source data pipe" that receives data from "dispatch process" right? 

In the dispatch process, the "sink data pipe" is actually the round_robin_demux?

Reply:
Correct!

Reply:
in that case shall we call the variable name as `dispatch_process_dps`? -- since it's a "source/receiving data pipe" that receives data on worker process? shall we call it something like `worker_source_dps`(don't have a good name suggestion :( )

Reply:
`dispatch_consumer_dps`?